doc/nrf/release_notes.rst

```diff
@@ -21,6 +21,7 @@ This page is included only in the latest documentation, because it might contain
    :caption: Subpages:
 
    releases/release-notes-changelog
+   releases/release-notes-2.0.0
```

Comment:
Please add the following changes also:
Line number 13: aspects ->aspect
Line number 15: remove this bullet point
(ref:https://github.com/nrfconnect/sdk-nrf/pull/6812/files#diff-243c4eb8fdad0e7a9c43c0cff444883d856cae37df3bc19e807c3db5ef90ac43R13)

Reply:
done